style.css

```diff
@@ -0,0 +1,7 @@
+/* .table {
+  display: none;
+} */
```

Comment:
- [ ] Please, kindly delete this old piece of commented code, to improve the cleanliness of the repo 💪